package.json

```diff
@@ -5,23 +5,25 @@
   "description": "A GitHub app for rebasing on demand",
   "author": "Balena",
   "license": "Apache-2.0",
-  "homepage": "https://github.com/balena-io-experimental/basejump",
+  "homepage": "https://github.com/product-os/basejump",
   "keywords": [
     "probot",
     "github",
     "probot-app"
   ],
   "scripts": {
     "clean": "rimraf lib",
-    "build": "npm run clean && tsc --project ./tsconfig.release.json",
+    "build": "tsc --project ./tsconfig.release.json",
+    "rebuild": "npm run clean && tsc --project ./tsconfig.release.json",
     "lint": "balena-lint -e ts src test",
     "lint:fix": "balena-lint --fix -e ts src test",
     "start": "probot run ./lib/index.js",
     "test": "npm run build && vitest",
-    "prepare": "husky"
+    "prepare": "node -e \"try { (await import('husky')).default() } catch (e) { if (e.code !== 'ERR_MODULE_NOT_FOUND') throw e }\" --input-type module"
   },
   "dependencies": {
-    "probot": "^13.4.3"
+    "probot": "^13.4.3",
+    "typescript": "^5.3.3"
```

Comment:
TS should arguably be a devDependency, I see why you put it here because of the Dockerfile though.

Reply:
Is there a better way we should solve that you think?

Reply:
A multistage Docker build, I would think